vtex/billing-app
# Post zero-valued contracts on free apps

#### What is the purpose of this pull request?
Create conditions for free apps to have contracts in Billing

#### What problem is this solving?
The `my-apps` cannot list not uninstalled free apps, because once this happens, they are not listed anywhere visible. By adding this feature, free apps purchases from now on will be recorded as Billing contracts, make them visible from the `contracts` query.

#### How should this be manually tested?
1. Clone/pull this repo/branch and link in your workspace
2. Purchase a free app, such as `vtex.my-orders-app` from the command line:
```shell
$ vtex install vtex.my-orders-app -r vtex --verbose
```
Or from the App Store:
https://apps.vtex.com/my-orders-app/r
3. Once this happens, the contract should be available in `https://{{account}}.myvtex.com/admin/billing/contracts`
4. Subsequent installs will consider the existence of this contract

#### Screenshots or example usage
![image](https://user-images.githubusercontent.com/3827456/40999031-0b33a07a-68e0-11e8-93b7-a185cdbcab2b.png)

#### Types of changes
